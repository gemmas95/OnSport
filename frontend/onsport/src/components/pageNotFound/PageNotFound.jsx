import React from 'react';
import './pageNotFound.scss';

function PageNotFound() {
    return (
        <section className="containerComponent">
            <h2>Ooops!!! Page Not Found</h2>
            <h3>Error 404</h3>
            <p>Nothing to do here!</p>
        </section>
    );
}

export default PageNotFound;
