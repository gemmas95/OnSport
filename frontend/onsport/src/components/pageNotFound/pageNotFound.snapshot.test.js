import renderer from 'react-test-renderer';
import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import PageNotFound from './PageNotFound';

describe('PageNotFound', () => {
    const pageNotFoundTree = renderer.create(
        <Router>
            <PageNotFound />
        </Router>
    );
    it('should match', () => {
        expect(pageNotFoundTree).toMatchSnapshot();
    });
});
