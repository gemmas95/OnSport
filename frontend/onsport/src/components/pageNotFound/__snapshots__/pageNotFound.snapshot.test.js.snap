// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound should match 1`] = `
<section
  className="containerComponent"
>
  <h2>
    Ooops!!! Page Not Found
  </h2>
  <h3>
    Error 404
  </h3>
  <p>
    Nothing to do here!
  </p>
</section>
`;
